index.html

```diff
@@ -1,203 +1,349 @@
+                        <div class="item-description">
+                            <div class="item-title">
+                                <h1>Mult-Post Stories</h1>
+                                <h2>Gain + Glory</h2>
+                            </div>
+                            <ul class="technologies">
+                                <li>Javascript</li>
+                                <li>Ruby on rails</li>
+                                <li>HTML</li>
+                                <li>CSS</li>
+                            </ul>
+                        </div>
+                        <div class="green-btn">
+                            <a href="single-project.html">See Project</a>
+                        </div>
                     </div>
+
                 </div>
 
-                <div class="project-grid-item">
-                    <div class="item-image">
-                        <!-- <img src="" alt="place image here" /> -->
-                    </div>
-                   <div class="item-description">
-                    <div class="item-title">
-                        <h1>Mult-Post Stories</h1>
-                        <h2>Gain + Glory</h2>
-                    </div>
-                    <ul class="technologies">
-                        <li>Javascript</li>
-                        <li>Ruby on rails</li>
-                        <li>HTML</li>
-                        <li>CSS</li>
-                    </ul>
-                   </div>
+            </section>
+            <section class="section">
+                <div class="about-bg-imgs">
+                    <img src="assets/images/about-me/about-me-left.svg" alt="bg-image"/>
+                    <img src="assets/images/about-me/aboutmedesktop-right.svg" alt="bg-image" />
+               </div>
+                <div class="about-header">
+                    <div class="title">
+                        <h1 class="primary-text">About Me</h1>
+                        <hr class="line" />
+                    </div>
+                    <p class="about-text">
+                        I can help you build a product , feature or website Look
+                        through some of my work and experience! If you like what
+                        you see and have a project you need coded,
+                        don’t hestiate to contact me.
+                    </p>
                     <div class="green-btn">
-                        <a href="" >See Project</a>
+                        <a href="">Get My Resume</a>
                     </div>
                 </div>
+                <div class="grid-container skills-set">
+                    <div class="grid-item skill-item">
+                        <div class="item-icon">
+                            <img src="assets/images/lang-icon.svg" alt="icon" />
+                        </div>
+                        <h3 class="item-title">Languages</h3>
+                        <ul class="item-langs">
```

Comment:
- [ ] Please use semantic HTML. Since you are making a list of all your skill-set, use a `<ul>` tag instead of a `<div>`. Each skill item should be a list item `<li>` 😎